Classes/Event/EventTypeResolver.php

```diff
@@ -132,12 +132,19 @@ public function getEventClassNameByType(string $eventType):string
     public static function eventTypeMapping(ObjectManagerInterface $objectManager)
     {
         $buildEventType = function ($eventClassName) use ($objectManager) {
-            $packageKey = $objectManager->getPackageKeyByObjectName($eventClassName);
-            if ($packageKey === false) {
-                throw new Exception(sprintf('Could not determine package key from object name "%s"', $eventClassName), 1478088597);
+            $eventReflection = new \ReflectionClass($eventClassName);
+            if ($eventReflection->hasProperty('eventType')) {
```

Comment:
Maybe we should leave that particular change out of this PR as it's not directly related and create a separate one covering #94